includes/Story_Post_Type.php

```diff
@@ -574,6 +604,31 @@ public function get_editor_settings() {
 		return apply_filters( 'web_stories_editor_settings', $settings );
 	}
 
+	/**
+	 * Setup up post lock.
+	 *
+	 * @since 1.5.0
+	 *
+	 * @param int $story_id Post id of story.
+	 *
+	 * @return void
+	 */
+	protected function setup_lock( $story_id ) {
+		if ( current_user_can( 'edit_post', $story_id ) ) {
+			// Make sure these functions are loaded.
+			if ( ! function_exists( 'wp_check_post_lock' ) || ! function_exists( 'wp_set_post_lock' ) ) {
+				require_once ABSPATH . 'wp-admin/includes/post.php';
+			}
```

Comment:
Why shouldn't this function already exist? 🤔 

We're already in the admin and in the post editor, it should exist at this point, no?

If it's for tests, we could load the file in the test suite instead.

Reply:
I do this elsewhere in the core, just being overly careful. I can remove if it really bothers you. Just worried this function might be called out of context for some reason...